RELEASE-NOTES.md

```diff
@@ -1,5 +1,12 @@
 # Wikibase DataModel Serialization release notes
 
+## 3.0.0 (2017-02-16)
+
+* Changed the `DeserializerFactory` constructor to expect a list of Entity deserializers
+* Changed the `SerializerFactory` constructor to expect a list of Entity serializers
+* `DeserializerFactory::newEntityDeserializer` does not return default Item and Property deserializers any more
+* `SerializerFactory::newEntitySerializer` does not return default Item and Property serializers any more
```

Comment:
Speaking as maintainer of https://github.com/JeroenDeDauw/JsonDumpReader, I'm not happy with those changes. They make using this library harder for me and do not provide any benefit to me either. Why should I make updates to my code to accommodate features for Wiktionary and Commons that are not relevant to my tool that just needs to deal with items and properties?